src/Adapter/Category/CommandHandler/EditCategoryHandler.php

```diff
@@ -53,6 +55,11 @@ public function handle(EditCategoryCommand $command)
             throw new CategoryNotFoundException($command->getCategoryId(), sprintf('Category with id "%s" cannot be found.', $command->getCategoryId()->getValue()));
         }
 
+        // Root category means here REAL ROOT category, not the HOME of the shop
```

Comment:
```suggestion
```

IMO useless comment as you did a good job with a method description :) if you think it needs to be even more clear to developers in future I'd suggest putting more details there